diff --git a/Cargo.lock b/Cargo.lock
index 8174a2b..b8dc65c 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -504,6 +504,7 @@ name = "chip"
version = "0.3.0"
dependencies = [
  "clap",
+ "gcl",
  "indexmap",
  "itertools 0.14.0",
  "lalrpop",
@@ -3146,7 +3147,6 @@ name = "tapi"
version = "0.1.0"
source = "git+https://github.com/oeb25/tapi.git#25c4068cef6aa56d0ef5fd0a00f9cda15879809b"
dependencies = [
- "axum",
  "chrono",
  "dyn-clone",
  "futures-util",
diff --git a/Cargo.toml b/Cargo.toml
index dcd5136..373df8f 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -80,7 +80,7 @@ stdx = { path = "./crates/stdx" }
# ] }
tapi = { git = "https://github.com/oeb25/tapi.git", features = [
     "chrono",
-    "endpoints",
+    # "endpoints",
     "smol_str",
     "toml",
] }
diff --git a/crates/chip/Cargo.toml b/crates/chip/Cargo.toml
index b99537c..99729e1 100644
--- a/crates/chip/Cargo.toml
+++ b/crates/chip/Cargo.toml
@@ -18,6 +18,7 @@ serde.workspace = true
smtlib.workspace = true
thiserror.workspace = true
tracing.workspace = true
+gcl.workspace = true

[build-dependencies]
lalrpop.workspace = true
diff --git a/crates/stdx/Cargo.toml b/crates/stdx/Cargo.toml
index da75ca1..5707065 100644
--- a/crates/stdx/Cargo.toml
+++ b/crates/stdx/Cargo.toml
@@ -12,5 +12,8 @@ once_cell.workspace = true
serde.workspace = true
serde_json.workspace = true
tapi.workspace = true
-tokio.workspace = true
+tokio = { workspace = true, optional = true }
tracing.workspace = true
+
+[features]
+tokip = ["tokio"]
diff --git a/crates/stdx/src/lib.rs b/crates/stdx/src/lib.rs
index 05eda70..d2eae6a 100644
--- a/crates/stdx/src/lib.rs
+++ b/crates/stdx/src/lib.rs
@@ -1,2 +1,3 @@
+#[cfg(feature = "tokio")]
pub mod concurrency;
pub mod stringify;
 